pandas/_libs/tslibs/conversion.pyx

```diff
@@ -572,6 +594,45 @@ cdef inline void localize_tso(_TSObject obj, tzinfo tz):
     obj.tzinfo = tz
 
 
+cdef inline bint _infer_tsobject_fold(_TSObject obj, object trans,
+                                      object deltas, int32_t pos):
```

Comment:
can trans or deltas be typed as e.g. ndarray[int64_t]?

Reply:
we type these in the localizaiton functions i think

Reply:
You are right, thanks. Typed trans and deltas.